docker/README.md

```diff
@@ -0,0 +1,46 @@
+## docker installation
+
+1. build the tradenote image with your custom parameters
+```
+docker build . -t tradenote --build-arg PARSE_INIT=APPLICATION_ID --build-arg PARSE_URL=http://pi:1337/parse
```

Comment:
I think it would be better to replace http://pi:1337/parse with PARSE_SERVER_URL/parse to make it more generic

Reply:
yes, your way is better